pom.xml

```diff
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.github.mwiede</groupId>
-    <artifactId>jsch</artifactId>
+    <artifactId>k2-jsch</artifactId>
```

Comment:
I think this was added by mistake right?

Reply:
Right. Currently we are using our fork in our code.